docs/topics/api/signing.rst

```diff
@@ -46,9 +46,11 @@ validation and you will be able to check its status.
             -g -XPUT --form 'upload=@build/my-addon.xpi'
             -H 'Authorization: JWT <jwt-token>'
 
-    :param addon-id the id for the add-on.
-    :param version: the version of the add-on.
-    :form upload: the add-on being uploaded.
+    :param addon-id: The id for the add-on.
+    :param version: The version of the add-on. A version ending with
+        a|alpha|b|beta and an optional number is detected as beta.
```

Comment:
That's probably a good idea, I don't really like the whole `a|alpha|b|beta` thing since it is supposed to be a regex I guess. I'll rewrite it in English since we have examples.